cPP_DSC.adoc

```diff
@@ -2598,9 +2610,9 @@ _There are several reasons for granting access to keys or restricting access to
 +
 _This SFR lists several methods for protecting the confidentiality of keys prior to authorizing their use, archival, backup, escrow, or recovery. This protection is often afforded only to the private or secret portion of cryptographic keys. Some methods offer integrity protection as well. These methods may not apply to modification of attributes especially if the attributes do not need confidentiality. These methods may not apply to destruction since a TOE can only destroy keys within its boundaries, and it is premused these keys are already in the boundary._
 +
-*_Key encapsulation* - If the STF claims key encapsulation, then it should use approved methods such as those listed in FCS_CKM.2._
+*_Key encapsulation* - If the STF claims key encapsulation, then it should use approved methods such as those listed in FCS_COP.1/KeyEncap._
 +
-*_Key wrapping* - If the STF claims key wrapping, then it should use approved methods such as KW, or KWP._
+*_Key wrapping* - If the STF claims key wrapping, then it should use approved methods such as those listed in FCS_COP.1/KeyWrap._
```

Comment:
Seeing as we have explicit SFRs to cover those cases, it seems sensible to refer to them rather than more vague SFRs (FCS_CKM.2) or no SFRs at all.